pkg/webhook/server/server.go

```diff
@@ -403,3 +422,67 @@ func (s *Server) handleLivez(w http.ResponseWriter, req *http.Request) {
 
 	w.WriteHeader(http.StatusOK)
 }
+
+// these conversions are copied from https://github.com/kubernetes/kubernetes/blob/4db3a096ce8ac730b2280494422e1c4cf5fe875e/pkg/apis/admission/v1beta1/zz_generated.conversion.go
+// to avoid copying in kubernetes/kubernetes
+// they are sightly modified to remove complexity
+
+func convert_v1beta1_AdmissionReview_To_admission_AdmissionReview(in *admissionv1beta1.AdmissionReview, out *admissionv1.AdmissionReview) {
```

Comment:
Move conversion functions to a sub module. Especially since we are importing unsafe.